DiscordChatExporter.Core/Exporting/ExportContext.cs

```diff
@@ -106,6 +106,18 @@ await foreach (var role in Discord.GetGuildRolesAsync(Request.Guild.Id, cancella
             .FirstOrDefault();
     }
 
+    public IEnumerable<Role> TryGetUserRoles(Snowflake id)
```

Comment:
Let's make this `IReadOnlyList<Role> GetUserRoles(...)` (no need for `Try...` because it doesn't return `null`).

Reply:
Also, let's update `TryGetUserColor(...)` so that it calls `GetUserRoles(...)` for code reuse